cmd/fs-v1.go

```diff
@@ -1246,6 +1246,11 @@ func (fs *FSObjects) HealBucket(ctx context.Context, bucket string, dryRun, remo
 	return madmin.HealResultItem{}, NotImplemented{}
 }
 
+// Walk - walks the bucket at prefix recursively until EOF.
+func (fs *FSObjects) Walk(ctx context.Context, bucket, prefix string, doneCh chan struct{}) <-chan ObjectInfo {
```

Comment:
```suggestion
func (fs *FSObjects) Walk(ctx context.Context, bucket, prefix string, doneCh chan struct{}) <-chan ObjectInfo {
```

`done` is redundant. A context is already included which can be used for early termination.

Reply:
It isn't unfortunately we haven't used context in this manner. So I won't add it now.